lms/djangoapps/bulk_email/models.py

```diff
@@ -198,14 +198,12 @@ def short_display(self):
         return "{}-{}".format(self.target_type, self.track.mode_slug)  # pylint: disable=no-member
 
     def long_display(self):
-        all_modes = CourseMode.objects.filter(
-            course_id=self.track.course_id,
-            mode_slug=self.track.mode_slug,  # pylint: disable=no-member
-        )
-        return "Course mode: {}, Currencies: {}".format(
-            self.track.mode_display_name,  # pylint: disable=no-member
-            ", ".join([mode.currency for mode in all_modes])
-        )
+        course_mode = self.track
+        long_course_mode_display = 'Course mode: {}'.format(course_mode.mode_display_name)
+        if course_mode.mode_slug not in CourseMode.AUDIT_MODES:
+            mode_currencies = 'Currency: {}'.format(course_mode.currency)
```

Comment:
This really ought to be wrapped in a `gettext` call for internationalization, but since we weren't doing it before don't let it block this quick fix.

Slight nit: I notice you changed from the plural `Currencies` to the singular `Currency`; was that deliberate? Do we expect each course mode to have a single currency? If so, could you please also update the variable to be the singular `mode_currency` for consistency?

Reply:
@efischer19 --
```
I have not yet seen a case where a course has a course mode (say `audit`) with multiple currencies. Do you know any of such case? 

Going to change the var name to be singular.

Reply:
This made me nervous, so I double checked. The `course_modes_coursemode` table only has one currency column. And, it doesn't look like there are any cases where a course has multiple course modes of the same mode_slug:

```sql
mysql> select course_id, mode_slug from course_modes_coursemode group by course_id, mode_slug having count(*) > 1;
Empty set (0.08 sec)
```

It doesn't look like there are any cases where a course has two different course modes with the same slug (like `audit`) but different currencies.